scripts/audit_passing_way_junctions.py

```
@@ -0,0 +1,114 @@
+
+    targets = set()
+    with open(node_ids_path) as f:
+        for line in f:
+            line = line.strip()
+            if line.isdigit():
+                targets.add(int(line))
+
+    # node_id -> [way 本数, 実分岐数]
+    stats = {}
+
+    proc = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)